.github/ISSUE_TEMPLATE/membership.yml

```diff
@@ -0,0 +1,64 @@
+name: Consortium Membership
+description: Application to become a Safety Critical Rust Consortium Member
+title: Membership Application for [INSERT COMPANY, INSTITUTION, YOUR NAME, OR OTHER ENTITY]
+labels: [membership, 'status: needs review']
+assignees: 
+  - joelmarcey
+body:
+  - type: markdown
+    attribute:
+      value: |
+        The [Safety Critical Rust Consortium](https://github.com/rustfoundation/safety-critical-rust-consortium) was [announced](https://foundation.rust-lang.org/news/announcing-the-safety-critical-rust-consortium/) in June 2024 with a goal to support the responsible use of Rust in Safety Critical software. The mission of the consortium is to improve the suitability of Rust in safety critical applications.
+        
+        Membership into the consortium is free and available to those with a noted and vested interested in the safety critical space. There are two types of membership:
+        - Producer: An active participant in the consortium, helping drive its mission
+        - Observers: A passive participant who wants to keep up with the consortium's happenings. Observers generally virtually attend meetings and listen in.
+        
+        This distinction is mainly for housekeeping purposes. You can choose which type of membership you plan to be initially and change it as needed.
+
+        If you do become a member of the consortium, you will be added to the consortium mailing list, which is currently the only non-public asset of the consortium at this point.
+  - type: input
+    id: Name
+    attributes:
+      label: What is your name?
+    validations:
+      required: true
+  - type: dropdown
```

Comment:
Disclaimer: I haven't done a GitHub issue template before.

It seems like from the [docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#dropdown) for the `dropdown` input type it's expected to have an `options` value in the `attributes`:

```
body:
- type: dropdown
  id: download
  attributes:
    label: How did you download the software?
    options:
      - Built from source
      - Homebrew
      - MacPorts
      - apt-get
    default: 0
  validations:
    required: true
```

I see there are options listed up here: Company, Institution, and so on, but they are not under an `options` value.